cypress/support/fragments/orders/newOrder.js

```diff
@@ -22,6 +22,6 @@ export default {
   defaultOneTimeOrder: {
     id: uuid(),
     vendor: '',
-    orderType: 'One-Time'
+    orderType: 'One-time'
```

Comment:
Are you sure this change is needed here?

Reply:
yep